README.md

```diff
@@ -81,6 +81,52 @@ signing {
     sign(publishing.publications["mavenJava"])
 }
 ```
+
+#### Publishing with Ivy ####
+
+There are cases where it may be necessary to use the `ivy-publish` plugin instead of `maven-publish`.
+For example, when publishing Sbt plugins the directory structure needs to be customized which is only possible with Gradle's `IvyArtifactRepository`.
+
+In such cases, you need to apply the `ivy-publish` plugin and configure the `nexusPublishing` extension's `publicationType` to `IVY` (default is `MAVEN`).
+
+In case of Ivy publishing, because of compatibility with Sonatype the nexus repository layout will be used by default
+
+```groovy
+nexusPublishing {
+    publicationType = io.github.gradlenexus.publishplugin.NexusPublishExtension.PublicationType.IVY
+}
+```
+
+Or use the kotlin DSL:
+
+```kotlin
+nexusPublishing {
+    publicationType.set(io.github.gradlenexus.publishplugin.NexusPublishExtension.PublicationType.IVY)
```

Comment:
This needs to be updated now that `publicationType` is defined on `NexusRepository`